shared/qcommon/q_platform.h

```diff
@@ -227,11 +227,74 @@ along with this program; if not, see <http://www.gnu.org/licenses/>.
 #endif
 
 // endianness
-void CopyShortSwap( void *dest, void *src );
-void CopyLongSwap( void *dest, void *src );
-short ShortSwap( short l );
-int LongSwap( int l );
-float FloatSwap( const float *f );
+// Use compiler builtins where possible for maximum performance
+#include <stdint.h>
+#if defined(__clang__)
```

Comment:
Can this block be put in an `#else` block? There may be other compilers which we haven't tested with and there needs to be some fallback.